test/pub_sub_test.exs

```diff
@@ -14,8 +23,58 @@ defmodule PubSubTest do
     assert PubSub.subscribers(topic2) == [pid3]
   end
 
+  test "processes can unsubscribe from topics" do
+    [pid1, pid2, pid3] = spawn_multiple(3)
+    {topic1, topic2} = {:erlang, :elixir}
+    PubSub.subscribe(pid1, topic1)
+    PubSub.subscribe(pid2, topic1)
+    PubSub.subscribe(pid3, topic2)
+    PubSub.unsubscribe(pid2, topic1)
+
+    assert PubSub.subscribers(topic1) == [pid1]
+    assert PubSub.subscribers(topic2) == [pid3]
+  end
+
+  test "list of current topics can be retrieved" do
+    pid = spawn(fn -> receive do end end)
+    {topic1, topic2, topic3} = {:elixir, :erlang, :opensource}
+    PubSub.subscribe(pid, topic1)
+    PubSub.subscribe(pid, topic2)
+    PubSub.subscribe(pid, topic3)
+
+    assert PubSub.topics() == [:elixir, :erlang, :opensource]
+  end
+
+  test "process can publish a message to a topic" do
+    pid = spawn(fn ->
+      receive do
+        message ->
+          assert message == "Hello!"
```

Comment:
I don't believe this is being reached, unsure of how to assert that this subscriber received the message

Reply:
I think this is what you're looking for: https://hexdocs.pm/ex_unit/ExUnit.Assertions.html#assert_received/2